Model/Job.php

```diff
@@ -0,0 +1,117 @@
+<?php
+namespace Algolia\AlgoliaSearch\Model;
```

Comment:
Can you leave an empty line between `<?php` and `namespace`? Applies for all the files